buildConfiguration.xml

```diff
@@ -2,7 +2,7 @@
   <dotnetArchitecture>x64</dotnetArchitecture>
   <nugetVersion>3.5.0-rc-1285</nugetVersion>
   <runtimes>net45,net461,netstandard2.0</runtimes>
-  <assemblyVersion>6.12.1</assemblyVersion>
+  <assemblyVersion>6.12.3</assemblyVersion>
```

Comment:
What changes are in this version? Or is this preparing for the next version?

Reply:
We shipped 6.12.2, so we need to bump the version.